front-end/src/components/TextInput.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+
+export default function CommonInput(props) {
```

Comment:
This component should be called TextInput or this file should be called CommonInput.js. Otherwise the naming is inconsistent.

Reply:
same for some other files